addons/udes_stock/models/stock_picking_type.py

```diff
@@ -78,6 +78,12 @@ class StockPickingType(models.Model):
         'scan something from pickings of this type'
     )
 
+    u_scan_parent_package_end = fields.Boolean(
+        string='Scan parent package at the end',
+        default=False,
+        help='If True, the user is asked to scan parent package on drop off.',
```

Comment:
Not true. That would depend on `u_user_scans` and `u_target_storage_format`. So, having such boolean option is misleading.

As discussed, we should move such logic from the frontend. And use an enumeration for the 3 possible cases (don't scan, scan at start, scan at the end). We should at least add a TODO here.